vcpkg/contributing/maintainer-guide.md

```diff
@@ -50,16 +52,26 @@ then obviously beneficial changes like fixing typos are appreciated!
 
 ### Check names against other repositories
 
-A good service to check many at once is [Repology](https://repology.org/).
-If the library you are adding could be confused with another one,
-consider renaming to make it clear. We prefer when names are longer and/or
-unlikely to conflict with any future use of the same name. If the port refers
-to a library on GitHub, a good practice is to prefix the name with the organization
-if there is any chance of confusion.
+Port names should, as much as possible, be unambiguous about the package the port
+installs. When a specific name is strongly associated with a single project the
+port can simply take the name of the package. For example, `openssl`, `tensorflow`,
+and `zlib`.
 
-Put another way, the reason for this is to ensure that `vcpkg install Xxx`
-gives the user looking for `Xxx` what they were expecting and not be
-surprised by getting something different.
+Some times a package may use a common word as the package's name, especially if
+there are no projects with a strong association to the given word. For example,
+a package may want to call itself `print`. They may add a suffix or prefix like
+`libprint`, `openprint`, `print2` or `print-cpp` to the name. In such cases, contributors
+should prepend the organization name or author name to the port name to make the
+resolution unambiguous: `username-print`.
+
+When deciding the port name of a given package, prefixes and suffixes should not
+be considered as differentiators. Instead, the "stem" of the package should be

```

Comment:
This should give an "including but not limited to" list of what kind of prefixes and suffixes are ignored:

- Number suffixes (`curl2` is similar to `curl`)
- Lib prefix (`libcurl` is similar to `curl`)
- Open prefix (`opencurl` is similar to `curl`)

The consequence of comparing the stem is that projects will be evaluated _as though_ they were being named just the stem -- so `libspacebanana` should be evaluated as just `spacebanana` for the purposes of other guidelines (which is still fine).